packages/trader/src/Modules/Trading/Containers/trade-params.jsx

```diff
@@ -11,28 +12,43 @@ import TakeProfit from 'Modules/Trading/Components/Form/TradeParams/Multiplier/t
 import Expiration from 'Modules/Trading/Components/Form/TradeParams/Multiplier/expiration.jsx';
 import { connect } from 'Stores/connect';
 
-const TradeParams = ({ form_components, is_minimized }) => {
+const TradeParams = ({ form_components, is_minimized, is_turbos }) => {
+    const [is_barriers_table_expanded, setIsBarriersTableExpanded] = useState(false);
+
     const isVisible = component_key => {
         return form_components.includes(component_key);
     };
+
+    const toggleBarriersTable = e => {
+        e.preventDefault();
+        setIsBarriersTableExpanded(!is_barriers_table_expanded);
```

Comment:
Just to be cautious: maybe it'll be better to set `true` or `false` instead of using `!` ?